File: frontend/lib/models/announcement_model.dart
class Announcement {
  Announcement({
    required this.rtId,
    required this.title,
    required this.message,
    required this.isBoysHostelRt,
    required this.blockNo,
    required this.announcementId,
    required this.isRead,
  });
  final String rtId;
  final String announcementId;
  final bool isBoysHostelRt;
  final bool isRead;
  final String title;
  final String message;
  final int blockNo;
}

File: backend/models/announcement_model.js
const mongoose = require('mongoose');

const announcementSchema = mongoose.Schema({
    title:{type:String,required:true},
    message:{type:String,required:true},
    rtId:{type:String,required:true},
    blockNo:{type:Number,required:true},
    isRead:{type:Boolean,required:true,default:false},
    isBoysHostelRt:{type:Boolean,required:true}
})

module.exports = mongoose.model('announcement',announcementSchema);